test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -1092,4 +1102,104 @@ public void forceRemoveAfterSeal() throws Exception {
+        final Duration timeout = Duration.ofMinutes(5);
+        final Duration pollPeriod = Duration.ofSeconds(5);
+
+        final UUID streamId = CorfuRuntime.getStreamID(streamName);
+        final String checkpointStreamName = CorfuRuntime.getStreamID(streamName).toString()
+                + StreamsView.CHECKPOINT_SUFFIX;
+        final UUID checkpointStreamId = UUID.nameUUIDFromBytes(checkpointStreamName.getBytes());
+
+        final int numCheckpointRecordsDefault = 3;
+
+        Process server0 = runPersistentServer(corfuSingleNodeHost, PORT_0, true);
+        runtime = createDefaultRuntime();
+
+        CorfuTable<String, String> table = runtime.getObjectsView().build()
+                .setTypeToken(new TypeToken<CorfuTable<String, String>>() {})
+                .setStreamID(streamId)
+                .open();
+
+        final int numEntries = 3;
+        for (int i = 0; i < numEntries; i++) {
+            table.put("k" + i, "v" + i);
+        }
+
+        // Checkpoint and trim the stream, no further updates on this stream (subsumed by checkpoint).
+        MultiCheckpointWriter mcw = new MultiCheckpointWriter();
+        mcw.addMap(table);
+        Token token = mcw.appendCheckpoints(runtime, "author");
+        runtime.getAddressSpaceView().prefixTrim(token);
+
+        // Add two other servers that triggers state transfer.
+        Process server1 = runPersistentServer(corfuSingleNodeHost, PORT_1, false);
+        runtime.getManagementView().addNode(getServerEndpoint(PORT_1), numRetry, timeout, pollPeriod);
+
+        Process server2 = runPersistentServer(corfuSingleNodeHost, PORT_2, false);
+        runtime.getManagementView().addNode(getServerEndpoint(PORT_2), numRetry, timeout, pollPeriod);
+
+        runtime.invalidateLayout();
+        Layout layout = runtime.getLayoutView().getLayout();
+
+        // Kill the server with primary sequencer, which triggers sequencer failover.
+        assertThat(shutdownCorfuServer(server0)).isTrue();
+        waitForEpochChange(refreshedEpoch -> refreshedEpoch > layout.getEpoch(), runtime);
+
+        // Create a new runtime.
+        CorfuRuntime runtime2 = createRuntime(getServerEndpoint(PORT_1));
+
+        // Verify sequencer has the correct steam tail.
+        assertThat(runtime2.getLayoutView().getLayout().getPrimarySequencer()).isNotEqualTo(getServerEndpoint(PORT_0));
+        assertThat(runtime2.getSequencerView().query(streamId).getSequence()).isEqualTo(numEntries - 1);
+
+        CorfuTable<String, String> table2 = runtime2.getObjectsView().build()
+                .setTypeToken(new TypeToken<CorfuTable<String, String>>() {})
+                .setStreamID(streamId)
+                .open();
+
+        // Verify sequencer has correct address map for this stream (addresses and trim mark)
+        StreamAddressSpace addressSpace = runtime2.getAddressSpaceView()
+                .getLogAddressSpace()
+                .getAddressMap().get(CorfuRuntime.getStreamID(streamName));
```

Comment:
@annym You can directly use the `streamId` variable here.

Reply:
Done.